client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -0,0 +1,35 @@
+import { Gridicon } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import React from 'react';
+import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan.svg';
+import { getTitanProductName } from 'calypso/lib/titan';
+import { ProviderCard } from 'calypso/my-sites/email/email-providers-stacked-comparison';
+
+import './professional-email-card.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+const logo = <Gridicon className="professional-email-card__logo" icon="my-sites" />;
+const badge = <img src={ poweredByTitanLogo } alt={ translate( 'Powered by Titan' ) } />;
+const getTitanFeatures = () => {
+	return [
+		translate( 'Inbox, calendars, and contacts' ),
+		translate( '30GB storage' ),
+		translate( '24/7 support via email' ),
+	];
+};
+
+export const professionalEmailCard: ProviderCard = {
+	detailsExpanded: true,
+	expandButtonLabel: translate( 'Expand' ),
+	formFields: <form> Placeholder </form>,
+	formattedPrice: '3.5$',
```

Comment:
I am not sure if these props should come from here, as they need to be computed dynamically based on the user's currency and need to wait for the query responses from the server.

Reply:
Yes, all those price tags has been addressed in the next pull request. Those were only placeholders :)